ports/gdal/vcpkg.json

```
@@ -1,9 +1,10 @@
 {
   "name": "gdal",
   "version-semver": "3.4.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The Geographic Data Abstraction Library for reading and writing geospatial raster and vector data",
   "homepage": "https://gdal.org",
+  "license": null,
```

```suggestion
  "license": "MIT",
```
`MIT style`: https://gdal.org/license.html.

---

Do read that file: "In general" MIT but ... there is more, in
https://raw.githubusercontent.com/OSGeo/gdal/master/LICENSE.TXT
This is also the file which we install for `copyright`.
That's why I explicitly chose `null` which is for [other licenses for which one should "just read the copyright file" (e.g., Qt)](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/manifest-files.md#license).
In particular in the light of Poppler: gdal[poppler] results in GPL-2.0-only.